src/components/patch/PatchContainer.test.tsx

```diff
@@ -10,7 +10,19 @@ import { PatchContainer } from "./PatchContainer";
 
 describe("PatchContainer", () => {
 
-  const wrapper = enzyme.mount(<PatchContainer client={rest.EvergreenClient("", "", "", "", true)} />);
+  const checkState = jest.fn(() => {
```

Comment:
Since this is only used for 1 test, let's scope the variable to just the test

Reply:
Fixed (sort of)